app/views/presentations/_form.html.erb

```diff
@@ -18,14 +18,7 @@
 
   <p>
     <%= f.label :event %><br>
-    <%= collection_select(:presentation, :event_id, Event.all, :id, :name, prompt: true) %>
-  </p>
-
-  <p>
-    <%= f.label :presenter %><br>
-    <%= collection_select(:presentation, :person_id, Person.all, :id, :first_name, include_blank: true) %>
-    or
-    <%= f.text_field :presenter, placeholder: 'Enter guest name' %>
+    <%= collection_select(:presentation, :event_id, Event.upcoming, :id, :name, prompt: true) %>
```

Comment:
That sounds like a good idea. Worth mentioning that as there is no edit action implemented yet I don't think there is a situation in which this form would appear that a presentation would already have an associated event already assigned.

However this form will surely be used when the edit functionality is introduced so I am going to include it. Cheers thanks!